src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -86,13 +103,20 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "md",
 .pf-c-description-list__term {
   font-size: var(--pf-c-description-list__term--FontSize);
   font-weight: var(--pf-c-description-list__term--FontWeight);
+  line-height: var(--pf-c-description-list__term--LineHeight);
 
   .pf-c-description-list__text {
     display: inline;
   }
 }
 
+.pf-c-description-list__description {
+  line-height: var(-pf-c-description-list__description--LineHeight);
+}
+
 .pf-c-description-list__text {
+  line-height: inherit;
```

Comment:
Is this necessary? `line-height` is an [inherited property](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#formal_definition), so whether it's used in `__term` or `__description`, it will inherit the `line-height` on its own.